src/processes/helper/preTopicAutoMsg.js

```diff
@@ -0,0 +1,35 @@
+const momentTz = require('moment-timezone');
+const {sample} = require('lodash');
+const {automateWelcomeMsgQueue} = require('../../config');
+const {LogError} = require('../../databases/models');
+
+const preTopicAutoMsgProcess = async (returnPrepopulatedDm) => {
+  try {
+    let currentTime = momentTz().tz('America/Los_Angeles');
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to hardcode timezone like 'America/Los_Angeles'. It would be better to move this to a configuration file or environment variable so that it can be easily changed if needed.
```diff
- let currentTime = momentTz().tz('America/Los_Angeles');
+ let currentTime = momentTz().tz(process.env.DEFAULT_TIMEZONE);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->